__tests__/test.js

```diff
@@ -151,36 +137,29 @@ describe('ol-mapbox-style', function() {
     });
 
     it('handles vector sources from TileJSON', function(done) {
-      var style_url = 'https://rawgit.com/PetersonGIS/CamoStyle/b783aadd625bf0d874f77daa6c597b585f0b63fd/camo3d.json';
```

Comment:
I added it as a local fixture and modified the test to use that instead of downloading it.  The test is still there.  It didn't seem that the test needed to connect to the outside world.  Does it?

Reply:
Thanks, I found the test. A local fixture works fine.